changelogs/unreleased/gh-9235-assertion_in_box_wait_limbo_acked.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/core
+
+* Fixed a bug when the assertion in box_wait_limbo_acked would fail.
+  The assertion is that the lsn of the last entry in limbo is always
+  positive after wal_sync (gh-7235).
```

Comment:
```suggestion
* Fixed a bug when the assertion in `box_wait_limbo_acked` would fail.
  The assertion is that the lsn of the last entry in limbo is always
  positive after `wal_sync` (gh-7235).
```